aunt-leahs-app/src/components/TableComponent.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import '../styles.css';
+import { Table, TableBody, TableCell, Paper, TableHead, TableRow } from '@material-ui/core';
+
+// REQUIRES: data prop has to be array of objects where each object key represents the name of the field in camelcase.
+//					 Refer to data in constants.jsx
+//					 each field in data should be either string or boolean. boolean fields are displayed '✓' or '✗'
+
+function header(title) {
```

Comment:
Function names should describe what the function _does_. Try adding a verb to the function name and see this link for more info: https://www.robinwieruch.de/javascript-naming-conventions

Reply:
I changed the name!